src/API.jl

```diff
@@ -81,6 +81,7 @@ function add(ctx::Context, pkgs::Vector{PackageSpec}; kwargs...)
     end
 
     ctx.preview && preview_info()
+    Types.update_registries(ctx)
```

Comment:
Did this happen further down in the call chain before?

Reply:
Yes, it was in `handle_repos_add`. It was the first call and we always call `handle_repos_add`. I thought it better if it was more clear where we were updating the registry